docs/source/api/apollo-server.mdx

```diff
@@ -599,6 +599,69 @@ The `start` method triggers the following actions:
 1. If your server is a [federated gateway](https://www.apollographql.com/docs/federation/managed-federation/overview/), it attempts to fetch its schema. If the fetch fails, `start` throws an error.
 2. Your server calls all of the [`serverWillStart` handlers](../integrations/plugins-event-reference/#serverwillstart) of your installed plugins. If any of these handlers throw an error, `start` throws an error.
 
+
+## `executeOperation`
+
+The async `executeOperation` method is used primarily for [testing GraphQL operations](../testing/testing/#testing-using-executeoperation) through the server's request pipeline without going through a full HTTP operation.
+
+The `executeOperation` method takes two arguments:
+* The first argument is a configuration options object
+* The second argument will be passed in to the current `ApolloServer` instance's `context` function
+
+Below are the available configuration options for the first argument of `executeOperation`:
+
+<table class="field-table">
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+<tr class="required">
+<td>
+
+##### `query`
+
+`String` or `<DocumentNode>`
```

Comment:
Ah yeah I haven't overhauled all the API references yet, but we should use lowercase `string` in JS/TS type references like here

Reply:
Ah and no angle brackets around `DocumentNode`, that's a nifty bit of TypeScript syntax called generics: https://www.typescriptlang.org/docs/handbook/2/generics.html

Wherever you see `Array<DocumentNode>` or similar in this document, that effectively means "an array where every element inside the array is a `DocumentNode`".  This is just one standalone `DocumentNode`.

Reply:
>  but we should use lowercase string in JS/TS type references like here

@StephenBarlow Would that also extend to object types? (`Object` -> `object`)

Reply:
Oh golly this is surprisingly tricky...let's go with "yes". And I bet we currently don't adhere to that everywhere so be prepared to spot inconsistencies 😄